interview-questions.md

```diff
@@ -8,42 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: Only difference I see between the two are naming convention and purpose. REACTs State for example, is a small piece of a program chained to others. OOB has built in functions that only operates as a function, such as console.log. 
 
-Researched answer:
+Researched answer: In Object oriented programming, objects hold data about them in attributes. The attributes are manipulated through functions or methods that are given to object. Functional programming should always have the same outputs given the same inputs.
 
 2. What is the difference between a Float and an Integer in Ruby?
 
-Your answer:
+Your answer: Integer is a numerical data type. A Float is the numerical data with decimals.
 
-Researched answer:
+Researched answer: Each number type is associated with a number class and can manipulated using methods. Floats and Integers each are umber classes converted using methods. 
 
 3. Ruby has an implicit return. What does that mean?
+No need for a return statement. In Ruby, the return is automatically implied.
 
-Your answer:
+Your answer: The last set of instructions asked will be executed in the program.
 
 Researched answer:
 
 4. What is a block in Ruby?
+Ruby is an object-oriented programming language (OOP) that uses classes as blueprints for objects. Objects are the basic building-blocks of Ruby code (everything in Ruby is an object).
```

Comment:
I would recommend removing this as this looks like it is pulled straight from resources and is not specific to this question.